docs/PrismaMigrationTools.md

```diff
@@ -0,0 +1,49 @@
+# Prisma Migration Tools
+
+## `npm prisma:seed`
+
+### What it does
+
+This command seeds the database, which means populating the database with initial data it requires to run. It can be dummy data, basic user accounts, a default language, etc.
+
+### Why to use it
+
+While developing the application, we need some data to be present. Running this command populates the database with the data it needs.
```

Comment:
I think we can be more specific. It needs data in order to have anything show up on the front end to interact with. It will populate the database with a basic set of data to use for development purposes.